Vagrantfile

```diff
@@ -20,16 +20,22 @@ DISTRO=$(lsb_release -is | tr '[:upper:]' '[:lower:]')
 CODENAME=$(lsb_release -cs)
 echo "deb http://repos.mesosphere.io/${DISTRO} ${CODENAME} main" | sudo tee /etc/apt/sources.list.d/mesosphere.list
 
-apt-get -q -y update
+apt-get -o Acquire::http::Timeout=1 -o Acquire::ftp::Timeout=1 -y update
```

Comment:
You mention this timeout change in the description:   lower timeout value for apt-get. This helps avoid long pause times when "apt-get update" is run thereby improving the time that it takes to build a docker image."

Is it safe/correct to do that?  i.e., if the timeout occurs then presumably it will break the resultant image?